front/src/reducers/osrdsimulation/selectors.ts

```diff
@@ -0,0 +1,10 @@
+import { RootState } from 'reducers';
+
+export const getAllowancesSettings = (state: RootState) => state.osrdsimulation.allowancesSettings;
+export const getMustRedraw = (state: RootState) => state.osrdsimulation.mustRedraw; //should disappear
+export const getPositionValues = (state: RootState) => state.osrdsimulation.positionValues;
+export const getSelectedProjection = (state: RootState) => state.osrdsimulation.selectedProjection;
+export const getSelectedTrain = (state: RootState) => state.osrdsimulation.selectedTrain;
+export const getTimePosition = (state: RootState) => state.osrdsimulation.timePosition;
+export const getConsolidatedSimulation = (state: RootState) => state.osrdsimulation.consolidatedSimulation;
+export const getPresentSimulation = (state: RootState) => state.osrdsimulation.simulation.present;
```

Comment:
you can use the util I have made

![Screenshot_20230127_111804](https://user-images.githubusercontent.com/24652545/215062654-215d066c-0009-4c05-9ffe-dabb22dafbc9.png)

Reply:
Done except from simulation.present